int64 max_num_candidates        # Maximum number of output poses (Results may contain a fewer number of poses than the requested number)
sensor_msgs/PointCloud2 cloud   # Some engines (e.g., BBS) assume that the cloud is gravity-aligned
---
std_msgs/Header header
std_msgs/Header globalmap_header

float64[] inlier_fractions
float64[] errors
geometry_msgs/Pose[] poses